lib/client/browser.js

```diff
@@ -238,6 +238,16 @@ require("./discovery.js");
 							testResult.success = false;
 							testResult.errors.push(msg);
 						}
+						if (config.logAssertions) {
+							const result = {
+								description: details.message,
+								suite: details.module && [qunitHtmlFile, details.module, details.name, details.message] || [],
+								success: details.result,
+								log: testResult.errors || [],
```

Comment:
I wonder whether the log of each assertion should contain the full errors array with all failed assertions from the current test.
I would expect this only to be the current one that has been pushed to the array (see `msg` in line 239).

Reply:
Didn't think of that. Let me fix it.